app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
+
+/*navbar*/
+
+#headWrapper{
+  height: 60px;
+  border-bottom: 2px solid;
+  margin-bottom:30px;
+  background-color:#373d46;
+  color: #4cc5e2;
+}
+
+#headWrapper header {
+  height: 60px;
+  width: 960px;
+  margin: auto;
```

Comment:
true.
What I mean is that it's better to have a class that holds those css rules.
For example, lets call it container
```
.constainer {
   width: 960px;
   margin: 0 auto;
}
```
And for each element you want these rules to apply, simply add this class.
For example:
`<header class="container">...`